### ISSUE TITLE
Multilanguage - URL Language Code & SEO

### ISSUE THREAD

**Issue by joomleb** [joomla/joomla-cms#22013]
Hi guys, 
Example: Joomla 3.8.12 Multilanguage En/Es Site enabled. The URLs will be:

 - mydomain.com (En language with Plugin > System - Language Filter > Remove URL Language Code: Yes)
 - mydomain.com/es

 - mydomain.com/terms-and-conditions
 - mydomain.com/**es**/terminos-y-condiciones > mydomain.com/terminos-y-condiciones

As far as I know there isn't a way, right now, to remove the URL language code for all languages less than the home page (logically). Am I right ?
A real language site should have different article / category Alias URL titles depending on language I'm using, so I'll have always unique aliases.

**New feature request**:

**A** - Would be a good think to extend the "Remove URL Language Code:" function to all languages enabled, the default one and the others
**B** - and / or simply add another option to the plugin, something like:
  - Plugin > System - Language Filter > Remove URL Language Code default language: Yes / No (it mean included the home page)
  - Plugin > System - Language Filter > Remove URL Language Code other languages: Yes / No (it mean on all pages less than the home page)

What do you think about ?

**Comment by brianteeman** [comment]
What would you do if the alias is the same in two languages

mydomain.com/es/villa
mydomain.com/en/villa

**Comment by infograf768** [comment]
Indeed. It is totally impossible to take off the url language code for all content languages.
It can be done only for the default site language.
Closing.

EDIT: This was already explained to you on the forum 2 years ago...

**Comment by joomleb** [comment]
Hi guys, 
thanks for reply here and sorry for my delay. I saw that the ticket has been closed, but I want to answer anyway for education and respect to you who have taken the time to read my ticket.

@brianteeman 
A - the possibility that it can happen should be very low;
B - as a plugin option, if I switch on I take that risk;
C - when happen I can always manage (modify/change) manually the alias URL.
Considering these 3 thinks I opened this ticket.

@infograf768 
"_EDIT: This was already explained to you on the forum 2 years ago..._"
I am (and always I'm try to be) a polite person. I'm a joomler from its beginning and I always follow all the community rules.
  - I made a question two years ago: "_Do you think it is anything we can expect with the "New URL Routing" ?_"  adding a comment to a topic https://forum.joomla.org/viewtopic.php?f=711&t=917190#p3437181 in the Language forum during the "New URL Routing" development.
  - You gently answered me: "..._The new router does not touch at that aspect_..." - "..._there is no way to do what you want_..."
  - I came back on the same topic two years later: "_I want come back to this old topic now that SEO feature has been completely added_"  https://forum.joomla.org/viewtopic.php?f=711&t=965224
  - Someone has splitted my comment from the original comment and, logically, now, the real mean of my question is incomprehensible and lost.
  - @fcoulter has gently explaned me "..._Part of the way that such sites work is that each URL contains a parameter for the language. It can only be dropped for the default language because that can be assumed if there is no language parameter in the URL_..."  and added: "..._You can post the feature request here, but I think that you will get the same response_..."

So, sorry if I didn't understood your "explanation" before.
